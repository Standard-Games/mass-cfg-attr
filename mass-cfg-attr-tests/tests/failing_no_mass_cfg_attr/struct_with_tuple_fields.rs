#![deny(unused_must_use, deprecated)]
#![allow(path_statements)] // disable warning for test

#[must_use]
struct ThreeInts(
    #[deprecated]
    i16,
    #[deprecated = "testing key-value"]
    i8,
    #[deprecated(since = "0.1.0", note = "this is a more complex example")]
    i32,
);

fn main() {
    ThreeInts(
        1,
        2,
        3,
    );
}
