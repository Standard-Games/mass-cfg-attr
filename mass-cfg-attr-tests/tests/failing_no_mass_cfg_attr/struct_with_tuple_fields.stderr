error: unused `ThreeInts` that must be used
  --> tests/failing_no_mass_cfg_attr/struct_with_tuple_fields.rs:15:5
   |
15 | /     ThreeInts(
16 | |         1,
17 | |         2,
18 | |         3,
19 | |     );
   | |______^
   |
note: the lint level is defined here
  --> tests/failing_no_mass_cfg_attr/struct_with_tuple_fields.rs:1:9
   |
1  | #![deny(unused_must_use, deprecated)]
   |         ^^^^^^^^^^^^^^^
